### ISSUE TITLE
[Bug]: Still not fixed in 1.8.2 

### ISSUE THREAD

**Issue by VillainVibe** [TheosCreation/CreateCompression#8]
### Minecraft Version

1.16

### Mod Version

1.8.2

### Is Compressium Loaded

Loaded

### What happened?

I downloaded 1.8.2 from curseforge.
Decompressing Grass blocks still give grass as output.
![image](https://github.com/TheosCreation/CreateCompression/assets/161154905/4a983bf8-6c65-4f78-aaa5-ab89f1707813)

### latest.log

none

**Comment by VillainVibe** [comment]
Okay

**Comment by TheosCreation** [comment]
hello just installed 1.8.2 version of my mod on 1.18.2 and recipe is correct for me.

when I pushed my changes I tested all three versions and it was working so maybe check to see if you have the correct version of the mod installed. Prior to the 1.8.2 version the grass recipe is wrong.

this is me on 1.18.2
![image](https://github.com/TheosCreation/CreateCompression/assets/83748550/92f182d3-645a-4f88-af3d-5db6949fd8fa)
